pkg/networkutils/network_test.go

```diff
@@ -360,8 +358,8 @@ func TestSetupHostNetworkWithExcludeSNATCIDRs(t *testing.T) {
 
 	mockProcSys.EXPECT().Set("net/ipv4/conf/lo/rp_filter", "2").Return(nil)
 
-	var vpcCIDRs []*string
-	vpcCIDRs = []*string{aws.String("10.10.0.0/16"), aws.String("10.11.0.0/16")}
+	var vpcCIDRs []string
+	vpcCIDRs = []string{"10.10.0.0/16", "10.11.0.0/16"}
```

Comment:
Nit: `:=` (unless there's some stylistic reason to keep the declaration separate from initialisation here?)

Reply:
Yeah, `staticcheck` warned me about these. Should be fixed for sure :)